satpy/composites/__init__.py

```diff
@@ -300,6 +300,8 @@ def __call__(self, projectables, nonprojectables=None, **info):
         projectables = self.match_data_arrays(projectables)
         info = combine_metadata(*projectables)
         info["name"] = self.attrs["name"]
+        if "standard_name" in self.attrs:
+            info["standard_name"] = self.attrs["standard_name"]
```

Comment:
Should this do `info.update(self.attrs)` instead to take any metadata that may be defined on YAML?

Reply:
Oooh, yeah, true! Nice catch!

Reply:
Updated in https://github.com/pytroll/satpy/pull/3091/commits/3cc1c6f8cb4eb1b8504a035b8fbed7b90f20f824